cloudant-client/src/test/java/com/cloudant/tests/CloudantClientHelper.java

```diff
@@ -47,6 +48,8 @@
         try {
             //a URL might be supplied, otherwise use the separate properties
             String URL = System.getProperty("test.couch.url");
+            System.out.println(System.getProperties());
```

Comment:
If you think this and its counterpart on the line below are necessary for debugging can you make it a JUL style thing so we can turn it off if we want. Or if it was just overlooked delete it.

Reply:
I'd left some debugging in, especially for when test parameters weren't being passed in properly from gradle. This is no longer needed, so I've got rid of all of these in 96ff132